src/Form/ConfirmDeleteMediaAndFile.php

```diff
@@ -134,6 +134,15 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
           $file = File::load($target_id);
           if ($file) {
             if (!$file->access('delete', $this->currentUser)) {
+              // May not be allowed access because it is a default thumbnail for media type.
+              if ($field->getName() == 'thumbnail') {
+                $default_thumbnail_filename = $entity->getSource()->getPluginDefinition()['default_thumbnail_filename'];
+                $default_thumbnail_uri = \Drupal::config('media.settings')
```

Comment:
Should be using dependency injection instead of the `\Drupal` thing... up in the [`::create()`](https://github.com/Islandora/islandora/pull/854/files#diff-fcdf66c32910fe0266f46f1aecae820e4ad718099a0853ef5a5bb3a04343e05fR64-R76) method, to load up the config factory, and either pass it to the constructor to get the particular config object and then to do so inside the constructor, or to grab and pass the config object in the `::create()` method to the constructor.